src/components/LmsConfigurations/index.jsx

```diff
@@ -115,6 +129,11 @@ class LmsConfigurations extends React.Component {
           >
             <MoodleIntegrationConfigForm enterpriseId={this.props.enterpriseId} />
           </Collapsible>
+        </div>
+        <div
+          key="successfactors-form-link"
```

Comment:
The lack of this was causing the LMS items to not line up.

Reply:
*eye twitches*